### ISSUE TITLE
Output to CSV columns don't appear to align with column headers

### ISSUE THREAD

**Issue by JCron245** [yargalot/gulp-accessibility#17]
Output looks like this 

```
heading, issue, element, line, column, description
NOTICE,"WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.2",<title>Angular Materialize Base Project</title>,,,6,4,Check that the title element describes the documen....
```

The extra commas between the **element** and **line** columns causes there to be alignment problem. This might be a problem with AccessSniff and not the gulp implementation but I haven't gotten a chance to test out AccessSniff by itself

**Comment by JCron245** [comment]
I put in a pull Request to the AccessSniff project that should fix this. The column headers were simply missing a few column names.
